### ISSUE TITLE
Prepaid data plan (resets on data used instead of set date)

### ISSUE THREAD

**Issue by TeddySadcat** [itsdrnoob/DataMonitor#103]
### Checklist

- [X] I made sure that there are no similar feature requests - [open](https://github.com/itsdrnoob/DataMonitor/issues) or [closed](https://github.com/itsdrnoob/DataMonitor/issues?q=is%3Aissue+is%3Aclosed).
- [X] I have taken the time to fill in all the required details in an appropriate descriptive way. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.

### Describe the Feature you'd like

Another data plan option that does not reset on a set date, but resets when data usage reaches a certain point.

### Why do you want this feature?

I use a prepaid "season pass" style data plan, which if the data limit is reached, mobile internet access simply stop. My data plan does not abide to date resets and thus current options do not have a way where I can manually reset data usage to zero once my quota is used up.

**Comment by itsdrnoob** [comment]
> My data plan does not abide to date resets and thus current options do not have a way where I can manually reset data usage to zero once my quota is used up.

Valid point, but you can change the data plan by setting a new start/end date to reset the data usage.

**Comment by TeddySadcat** [comment]
That is true, but frankly a quick reset button or manual data use input a là Glasswire would be viable too, thank you for the heads up.
